pkgs/development/compilers/openjdk/openjfx/12.nix

```diff
@@ -87,24 +88,20 @@ in makePackage {
   postFixup = ''
     # Remove references to bootstrap.
     find "$out" -name \*.so | while read lib; do
-      new_refs="$(patchelf --print-rpath "$lib" | sed -E 's,:?${bootjdk}[^:]*,,')"
+      new_refs="$(patchelf --print-rpath "$lib" | sed -E 's,:?${openjdk11_headless}[^:]*,,')"
       patchelf --set-rpath "$new_refs" "$lib"
     done
-
-    # Test to make sure that we don't depend on the bootstrap
-    if grep -q -r '${bootjdk}' "$out"; then
-      echo "Extraneous references to ${bootjdk} detected" >&2
-      exit 1
-    fi
   '';
 
+  disallowedReferences = [ openjdk11_headless ];
+
   passthru.deps = deps;
 
   meta = with stdenv.lib; {
     homepage = http://openjdk.java.net/projects/openjfx/;
-    license = openjdk.meta.license;
+    license = openjdk12.meta.license;
     description = "The next-generation Java client toolkit.";
     maintainers = with maintainers; [ abbradar ];
-    platforms = openjdk.meta.platforms;
+    platforms = [ "i686-linux" "x86_64-linux" ];
```

Comment:
This could be a little bit smarter if you took ```lib.attrNames outputHashes```. It should still work for platforms, we just don't have hashes added for them.

Reply:
I'm not sure if things have gotten better in OpenJFX 12, but OpenJFX 11 requires more than just new hashes to build on other platforms. I'm not sure I like this approach anyway because it makes the code less clear for little benefit.